src/config/schema.ts

```diff
@@ -0,0 +1,272 @@
+import { JTDDataType } from "ajv/dist/types/jtd-schema"
+
+export type ConfigSchema = JTDDataType<typeof configSchema>
+
+const buildNumberConfigSchema = {
+  type: "object",
+  properties: {
+    minBuildNumber: { type: "number" },
+    lastBuildNumber: { type: "number" },
+  },
+  required: ["minBuildNumber", "lastBuildNumber"],
+  additionalProperties: false,
+}
+
+const accountLimitConfigSchema = {
+  type: "object",
+  properties: {
+    level: {
+      type: "object",
+      properties: {
+        1: { type: "number" },
+        2: { type: "number" },
+      },
+      required: ["1", "2"],
+      additionalProperties: false,
+    },
+  },
+  required: ["level"],
+  additionalProperties: false,
+}
+
+const rateLimitConfigSchema = {
+  type: "object",
+  properties: {
+    points: { type: "number" },
+    duration: { type: "number" },
+    blockDuration: { type: "number" },
+  },
+  required: ["points", "duration", "blockDuration"],
+  additionalProperties: false,
+}
+
+const lndConfig = {
+  type: "object",
+  properties: {
+    name: { type: "string" },
+    type: {
+      type: "array",
+      items: { enum: ["offchain", "onchain"] },
+      uniqueItems: true,
+    },
+    priority: { type: "number" },
+  },
+  required: ["name", "type", "priority"],
+  additionalProperties: false,
+}
+
+export const configSchema = {
+  type: "object",
+  properties: {
+    name: { type: "string" },
+    funder: { type: "string" },
+    buildVersion: {
+      type: "object",
+      properties: {
+        android: buildNumberConfigSchema,
+        ios: buildNumberConfigSchema,
+      },
+      required: ["ios", "android"],
+    },
+    hedging: {
+      type: "object",
+      properties: {
+        LOW_BOUND_RATIO_SHORTING: { type: "number" },
+        LOW_SAFEBOUND_RATIO_SHORTING: { type: "number" },
+        HIGH_SAFEBOUND_RATIO_SHORTING: { type: "number" },
+        HIGH_BOUND_RATIO_SHORTING: { type: "number" },
+        LOW_BOUND_LEVERAGE: { type: "number" },
+        LOW_SAFEBOUND_LEVERAGE: { type: "number" },
+        HIGH_SAFEBOUND_LEVERAGE: { type: "number" },
+        HIGH_BOUND_LEVERAGE: { type: "number" },
```

Comment:
if these are not env variables should keep camel case standard but... :thinking:  are we using this values? this seems to be more a dealer config than backend config

Reply:
probably not used any more. would probably make sense to look at all the value in the yaml to see if they're still used as part of this PR

Reply:
It does not appear to be used by the Galoy app, so I will remove it.